doc/source/whatsnew/v0.21.1.txt

```diff
@@ -39,7 +39,7 @@ Deprecations
 Performance Improvements
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
--
+- Added a keyword argument, `cache`, to :func:`to_datetime` that improved the performance of converting duplicate datetime arguments (:issue: `11665`)
```

Comment:
let's move to 0.22.0

Reply:
Additionally, there should be two backticks around <code>\`\`cache\`\`</code>, and the space between `:issue:` and the issue number should be removed.